compare_runs.py

```diff
@@ -0,0 +1,56 @@
+import filecmp
+import getopt
+import os
+import sys
+import subprocess
+
+
+def get_arguments(argv):
+    bc_file = ''
+    output_bc = ''
+    test_count = 2
+    flags = ''
+    logfile_name = 'logfile'
+    try:
+        opts, args = getopt.getopt(argv,"hi:o:n:f:l:",[])
+    except getopt.GetoptError:
+        print('compare_runs.py -i <input_bc_file> -o <output_bc_file> -n <re-run_counts> -f "<clang_run_flags>" -l "<log_file_name>"')
+        sys.exit(2)
+    print(opts)
+    for opt, arg in opts:
+        if opt == '-h':
+            print('compare_runs.py -i <input_bc_file> -o <output_bc_file> -n <re-run_counts> -f "<clang_run_flags>" -l "<log_file_name>"')
+            sys.exit()
+        elif opt == '-i':
+            bc_file = arg
+        elif opt == '-o':
+            output_bc = arg
+        elif opt == '-n':
+            test_count = int(arg)
+        elif opt == '-f':
+            flags = arg
+        elif opt == '-l':
+            logfile_name = arg
+            
+    return bc_file, output_bc, test_count, flags, logfile_name
+
+
+def main(argv):
+    bc_file, output_bc, test_count, flags, logfile_name = get_arguments(argv)
+    first_counter = 0
+    current_full_path = os.getcwd()
```

Comment:
are you sure this works? getcwd() will return the path from which the command is run, not the path to the script. Example: 
$ pwd
/home/some/path
$ ~/path/to/script.py
getcwd() will return /home/some/path
We want to get /fullpath/to/script/

Reply:
Wouldn't getting full-path to the files mentioned in script work?

Reply:
not for the *.so, because we're trying to get the path to it. Right? For other files definitely.